videoframegrid.py

```diff
@@ -68,6 +71,28 @@ def swap_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
```

Comment:
Thought about that but reached the conclusion that I thought it was more readable to have `if not self.fullscreen` first since the programs initial state is in **non-fullscreen** mode. Reasonable or BS? ^^ You decide 

I can swing it either way, so it's up to you